neutron_plugin/rbac_policy.py

```diff
@@ -90,7 +90,7 @@ def find_resource_to_apply_rbac_policy(ctx):
 
 def validate_found_resource(input_dict, found_resource):
     if found_resource:
-        for key in found_resource.keys():
+        for key in list(found_resource.keys()):
```

Comment:
I don't think you need to actually wrap this in a list even though futurize may claim that you need to.